js/gui.js

```diff
@@ -14,15 +15,17 @@ window.addEventListener("load", function()
   var inputOptionsPromise = new Promise(function (resolve) {
     // input your character here in the form, "src_url": "character_name",
     resolve({
-      "characters/sheep.svg": "Intelligent Sheep",
-      "characters/robo.svg": "Robo",
-      "characters/swammy.svg": "Swammy",
-      "characters/monster.svg": "Monster",
-      "characters/elephant.svg": "Elephant",
-      "characters/shadowman.svg": "Shadowman",
-      "characters/santa.svg": "Santa Clause",
-      "characters/detective.svg":"Agent X",
-      "characters/Geekomoji.svg": "Geekomoji",
```

Comment:
On both sides there are 9 characters. Please make it 10 on the right side or rename your pull request to "replace character xy with block character."